katsdpcontroller/test/test_master_controller.py

```diff
@@ -884,8 +884,8 @@ async def test_sdp_shutdown(self, rmock: aioresponses) -> None:
         await self.client.request('capture-init', 'product')
         reply, informs = await self.client.request('sdp-shutdown')
         self.assertEqual(reply[0], b'127.0.0.144,127.0.0.42')
-        poweroff_mock.assert_any_call(url1, data=None)
-        poweroff_mock.assert_any_call(url2, data=None)
+        poweroff_mock.assert_any_call(url1, headers={'X-Poweroff-Server': '1'}, data=None)
```

Comment:
Is it worth testing without the X-Poweroff-Server header set ?

Reply:
This is a mock, so I don't see what the point would be - it would just confirm that when we don't pass the header, the header isn't there.

It would be a sensible test on the poweroff_server repo, but that doesn't currently have any unit tests.

Reply:
Fair enough...